ui/app/rspack.config.ts

```diff
@@ -23,7 +23,6 @@ module.exports = {
   context: __dirname,
   output: {
     path: path.resolve(__dirname, './dist'),
-    publicPath: 'PREFIX_PATH_PLACEHOLDER/',
```

Comment:
mmm that would be annoying to remove it as it is necessary for the `prefix_api` things. See #2576

Reply:
and it doesn't seem it fixed the e2e issue @jgbernalp.

But what we could do, is to rollback to remove the upgrade of rspack so it's not blocking the merge of this PR and the merge of the plugin.

What do you think ?

Reply:
but removing the `PREFIX_PATH_PLACEHOLDER` is fixing the issue that `npm start` doesn't work anymore.

Probably a compromise would be to do `publicPath: isDev? '': 'PREFIX_PATH_PLACEHOLDER/'`

Reply:
Ok then it needs to be only when building, but not on dev.

Reply:
this missing prefix also caused also an issue with the favicon so the e2e tests failed. But it should be good now. I applied the publicPath for non dev environments.

Reply:
unfortunately it seems there is something else